l10n_it_account_stamp/i18n/it.po

```diff
@@ -0,0 +1,109 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+# 	* l10n_it_account_stamp
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-12-05 15:25+0000\n"
+"PO-Revision-Date: 2018-12-05 16:31+0100\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"Language: it\n"
+"X-Generator: Poedit 1.8.12\n"
+
+#. module: l10n_it_account_stamp
+#: code:addons/l10n_it_account_stamp/invoice.py:73
+#, python-format
+msgid "Missing account configuration for %s"
+msgstr "Configurazione mancante sui conti di ricavo/costo per il prodotto %s"
```

Comment:
O la stringa sorgente è poco esplicativa o la stringa tradotta lo è troppo. :-)

In genere bisognerebbe evitare di spiegare il significato di una stringa.
Nelle linee guida (link seguente) ci sono due punti in particolare che spiegano bene la cosa:

http://tp.linux.it/buona_traduzione.html
1) sezione "La forma → Tradurre, non spiegare"
2) sezione "Traduzione di termini tecnici → Tradurre, non spiegare"

Reply:
fatto